### ISSUE TITLE
WCAG 4.1.2: ARIA hidden element must not contain focusable elements (#content)

### ISSUE THREAD

**Issue by jwunderl** [microsoft/pxt-microbit#2289]
#### Issue
`ARIA hidden element must not contain focusable elements` ([aria-hidden-focus](https://dequeuniversity.com/rules/axe/3.2/aria-hidden-focus?application=msftAI))
#### Target application
[Microsoft MakeCode for micro:bit](https://makecode.microbit.org/#editor)
#### Element path
#content
#### Snippet
`<div id="content" class="ui dimmable full-abs" aria-hidden="true">`
#### How to fix
    Fix all of the following:
      Focusable content should be disabled or be removed from the DOM
      Focusable content should have tabindex='-1' or be removed from the DOM
#### Environment
Chrome version 75.0.3770.142
This accessibility issue was found using Accessibility Insights for Web 2.7.1 (axe-core 3.2.2), a tool that helps find and fix accessibility issues. Get more information & download this tool at [http://aka.ms/AccessibilityInsights](http://aka.ms/AccessibilityInsights).

when share screen is popped up, all elements behind modal

**Comment by abchatra** [comment]
Fixed
